recipe/run_test.py

```diff
@@ -0,0 +1,77 @@
+# Configure CuPy to use 1 GPU for testing
+import ctypes
+import os
+import platform
+import site
+import sys
+os.environ["CUPY_TEST_GPU_LIMIT"] = "1"
+
+def get_target_name():
+    plat = platform.processor()
+    if plat == "aarch64":
+        out = "sbsa-linux"
+    else:
+        out = f"{plat}-linux"
+    return out
+
+# Check CUDA_PATH is set
+cuda_path = os.environ.get('CUDA_PATH')
+assert cuda_path is not None
+print("CUDA_PATH:", cuda_path)
+
+# check _wheel.json exists
+# note that we don't rely on cupy.__file__ because we cannot import CuPy on Windows
+pypaths = site.getsitepackages()
+for cp in pypaths:
+    config_json = os.path.join(cp, 'cupy/.data/_wheel.json')
+    if not os.path.isfile(config_json):
+        continue
+    else:
+        print("_wheel.json is found:", config_json)
+        break
+else:
+    raise RuntimeError('_wheel.json is not found in the package')
+
+# dlopen the driver stub so that CuPy can be imported later
+# Note: This would make the actual tests fail on GPU CI!
+if sys.platform.startswith('linux') and '12' in os.environ.get('CONDA_OVERRIDE_CUDA', ''):
+    try:
+        stub = f"{os.environ['PREFIX']}/targets/{get_target_name()}/lib/stubs/libcuda.so"
```

Comment:
I might have made a mistake here, I used the host not build platform, but it's not capturable in the CF CI because tests were supposed to skip in the cross-compiling setting, why is it run here?

Reply:
Ah, I had forgotten the `test: native` part in `conda-forge.yml`